src/test/java/com/puppycrawl/tools/checkstyle/xpath/XpathMapperTest.java

```diff
@@ -583,6 +580,212 @@ public void testIdent() throws Exception {
         assertArrayEquals("Result nodes differ from expected", expected, actual);
     }
 
+    @Test
+    public void testIdentByValue() throws Exception {
```

Comment:
Should this and other tests below be `ByText`?

Reply:
done